src/implementations/vanilla/components/breadcrumb-core/README.md

```diff
@@ -0,0 +1,3 @@
+# Breadcrumb Core
+
+(Work in progress)
```

Comment:
Same as the the others, guess we should not consider it as a work in progress anymore...